src/Sema.zig

```diff
@@ -1070,9 +1070,9 @@ fn zirOpaqueDecl(
     const tracy = trace(@src());
     defer tracy.end();
 
-    const inst_data = sema.code.instructions.items(.data)[inst].pl_node;
-    const src = inst_data.src();
-    const extra = sema.code.extraData(Zir.Inst.Block, inst_data.payload_index);
+    // const inst_data = sema.code.instructions.items(.data)[inst].pl_node;
+    // const src = inst_data.src();
+    // const extra = sema.code.extraData(Zir.Inst.Block, inst_data.payload_index);
```

Comment:
By not commenting out the code, and using `_ = foo;` instead, when a developer hits the TODO compile error below, it will have already verified for them that the above untagged union access is correct, saving them from having to double check.

In general, don't comment out code, use `_ = foo` instead. In the case of unfinished code like here, it's nice documentation of where someone left off. In the case of finished code, it is useful information about what calculations are ultimately discarded.